apps/crn-frontend/src/network/working-groups/Outputs.tsx

```diff
@@ -64,28 +133,27 @@ const OutputsList: React.FC<OutputsListProps> = ({
       }
       userAssociationMember={userAssociationMember}
       workingGroupAssociation
+      exportResults={exportResults}
```

Comment:
I believe it's necessary to pass

```
draftOutputs={draftOutputs}
```

here the same way it was done for `apps/crn-frontend/src/network/teams/Outputs.tsx` so the right text appears :)